tests/Parameters/PrefetchDataParameterTest.php

```diff
@@ -71,10 +68,10 @@ public function testResolveWithoutExistingResult(): void
 
         $resolvedParameterPromise = $parameter->resolve($source, $args, $context, $this->createStub(ResolveInfo::class));
 
-        self::assertFalse($buffer->hasResult($args));
+        self::assertFalse($buffer->hasResult($source));
         self::assertSame([$source], $buffer->getObjectsByArguments($args));
         self::assertSame($prefetchResult, $this->deferredValue($resolvedParameterPromise));
-        self::assertTrue($buffer->hasResult($args));
+        self::assertFalse($buffer->hasResult($source));
```

Comment:
Why was this assertion changed?

Reply:
Because after resolving Deferred on src/Parameters/PrefetchDataParameter.php:62 I clear the buffer from resolved values to prevent memory leaks